src/drivers/gitlab.js

```diff
@@ -140,7 +140,11 @@ class Gitlab {
     try {
       await exec('nvidia-smi');
     } catch (err) {
-      gpu = false;
+      try {
+        await exec('cuda-smi');
+      } catch (err) {
+        gpu = false;
+      }
```

Comment:
what do you think of this for :apple: support?

could go back to
```suggestion
      gpu = false;
```

if you prefer...

Reply:
What is `cuda-smi` supposed to do? 🤔

Reply:
it's what `nvidia-smi` is called on (some?) :apple: systems https://github.com/tmux-plugins/tmux-cpu/issues/24

Reply:
But it's a third party tool (?)

Should we rely on that to detect CUDA?

Reply:
idk, the impression I got was that CUDA can be installed on a mac without an `nvidia-smi` binary but with a `cuda-smi` binary available. Would be nice to get confirmation though.